README.md

```diff
@@ -45,10 +45,10 @@ After you're finished please remove all the comments and instructions!
 
 # 📗 Table of Contents
 
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
+- [📖 About the Project](The project is in development state for Portfolio mobile first)
+  - [🛠 Built With](HTML CSS)
     - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
```

Comment:
- [ OPTIONAL ] "Great job on adding the README to your project based on the previous reviewer's suggestion. However, it's important to keep the original table of contents intact as it enables users to navigate the document quickly and easily. To ensure a seamless user experience, I recommend leaving the table of contents as is and only modifying the main content of the README. Keep up the good work!" :+1: 
- Please remove this part of the README since it is of no relevance to this project :+1: 
![image](https://user-images.githubusercontent.com/102699958/227717167-2faa04ae-68d4-4f95-9915-644f4534d100.png)